tests/test_ninja.py

```diff
@@ -16,3 +16,4 @@ def _run(program, args):
 
 def test_ninja_module():
     _run("ninja", ["--version"])
+    _run("python",["-m", "ninja", "--version"])
```

Comment:
This test is better as two, to increase isolation.

```suggestion
def test_ninja_module():
    _run("ninja", ["--version"])
    
def test_ninja_package():
    _run("python",["-m", "ninja", "--version"])
```

Also it seems `_run()` forces `.py` on the program name, not sure this test is covering exactly what's intended? The `__main__` module probably isn't being imported.

Reply:
Any idea about my second comment here?